tests/test_helper.tcl

```diff
@@ -538,7 +539,7 @@ proc print_help_screen {} {
         "--valgrind         Run the test over valgrind."
         "--durable          suppress test crashes and keep running"
         "--stack-logging    Enable OSX leaks/malloc stack logging."
-        "--accurate         Run slow randomized tests for more iterations."
+        "--accurate         Run slow randomized tests for more iterations, and slow time driven tests."
```

Comment:
i'm still on the fence back and forth about this. waiting to see how #9033 evolves in order to avoid repeated changes.
@yossigo if you can, please post your thoughts.
i.e. i'd like to find a way to make these time-driven tests an opt-in.
maybe tag them as `slow`, but also have the `-slow` skip tag defined by default when someone just runs the `./runtest`, and implicitly removed when someone adds `--accurate`.

Reply:
I am waiting for #9033
I see that it supports ignore specified tags, which is exactly what is needed.
But I can't find the comment now.